0045-lorawan-frequency-plan-selection.md

```diff
@@ -87,6 +87,9 @@ After steps 2 and 3, before proceeding to the next escalation, DeWi's LoRaWAN
 Committee may withdraw their proposal and reissue a new proposal at anytime.
 This would effectively restart the process at step 1.
 
+## Live Virtual Public Forum
+
```

Comment:
Is there a way for the DeWi Comittee/Lorawan to notify each hotspot in an effected region when they have made a decision/at the beginning of the 4 week public commentary?

Reply:
Formal Dissent Shall be determined by writing "I formally dissent" in the comments on the PR.